recipes/toolwrapper/meta.yaml

```diff
@@ -1,41 +0,0 @@
-{% set name = "toolwrapper" %}
```

Comment:
Please, do not touch others recipes, revert it please

Reply:
similar to the first concern with the changed `example/meta.yaml` this `toolwrapper/meta.yaml` doesn't exist in the current commit

Reply:
unless i'm missing something obvious (which i very well could be)